tests/kernel/ltp_setup_networking.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This module installs and setup services and other things needed for
+# networking part of the LTP (Linux Test Project).
+# Maintainer: Petr Vorel <pvorel@suse.cz>
+
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+
+sub install_services {
+    my $self = shift;
+
+    # utils
+    zypper_call("in expect iputils net-tools-deprecated telnet");
+
+    # clients
+    zypper_call("in mrsh-rsh-compat telnet");
+
+    # services
+    zypper_call("in bind dhcp-server finger-server nfs-kernel-server rdist rpcbind rsync telnet-server vsftpd xinetd");
+    for my $i (qw(nfsserver rpcbind vsftpd xinetd)) {
+        assert_script_run "systemctl enable $i.service";
+        assert_script_run "systemctl start $i.service";
+    }
+}
+
+sub setup_env_variables {
+    my $self      = shift;
+    my $conf_file = '/root/env-variables.sh';    # TODO: remove duplicity with run_ltp.pm
+    my $content;
+
+    $content = <<EOF;
+# ltp specific setup
+rlogin
+rsh
+rexec
+pts/1
+pts/2
+pts/3
+pts/4
+pts/5
+pts/6
+pts/7
+pts/8
+pts/9
+EOF
+    assert_script_run "echo \"$content\" >> '/etc/securetty'";
+
+    # xinetd (echo)
+    assert_script_run 'sed -i \'s/\(disable\s*=\s\)yes/\1no/\' /etc/xinetd.d/echo';
+    assert_script_run 'sed -i \'s/\(#\s*bind.*\)/\1\nbind = 0.0.0.0/\' /etc/xinetd.conf';
```

Comment:
I know q(), but some escapes are for sed syntax (\( \1), so it's IMHO more readable to use plain quotes (the only escape is for quotes, the rest is output for shell).